just-install.json

```diff
@@ -157,6 +157,20 @@
       },
       "version": "2.2"
     },
+    "dropbox": {
+      "installer": {
+        "kind": "custom",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
+          ],
+          "extension": ".exe"
+        },
+        "x86": "https://www.dropbox.com/download?type=full&build={{.version}}&plat=win"
+      },
+      "version": "3.12.6"
```

Comment:
Yes, that lightens the maintenance burden. We should pin to a specific version only if two versions can live side by side or there's widespread reported breakage with a particular version.

I'll update the contributing guidelines ASAP.